src/ui/public/vislib/__tests__/lib/layout/layout.js

```diff
@@ -12,6 +12,8 @@ import $ from 'jquery';
 import VislibLibLayoutLayoutProvider from 'ui/vislib/lib/layout/layout';
 import FixturesVislibVisFixtureProvider from 'fixtures/vislib/_vis_fixture';
 import PersistedStatePersistedStateProvider from 'ui/persisted_state/persisted_state';
+import VislibVisConfig from 'ui/vislib/lib/vis_config';
```

Comment:
Do you want to update these kinds of paths to be relative? I see some of the files have relative paths but some are still absolute like this one.